recipes/libcxxwrap-julia/bld.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D BUILD_TESTS=OFF -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+if errorlevel 1 exit 1
```

Comment:
Should we include this given we are not using it currently?

Reply:
I think so since the limitation is merely a dependency that has not been built for conda-forge.